ra/ra.go

```diff
@@ -1657,6 +1704,14 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 			continue
 		}
 		authz := nameToExistingAuthz[name]
+		// If the existing authz isn't valid, note that its missing and continue.
```

Comment:
It looks like this re-removes pending authz reuse. I think that's not intentional, right?

Reply:
You're correct - I think this was a merge mistake. I will fix

Reply:
Thanks for catching that. It would have been really confusing if it slipped by! Fixed in a2b1e1d